DataStructures/Graphs/Kruskal.java

```diff
@@ -0,0 +1,94 @@
+package Kruskal;
```

Comment:
The repository is for educational purpose and therefore it will be great if you could add a description of your problem.